client/src/test/java/io/pravega/client/security/auth/JwtTestUtils.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.security.auth;
+
+import com.google.gson.Gson;
+import io.pravega.test.common.JwtBody;
+import java.util.Base64;
+
+public class JwtTestUtils {
```

Comment:
There is a class by this name in `io.pravega.common`. May I suggest merging these methods in that one?

Reply:
Deleted this one